mslib/msui/editor.py

```diff
@@ -50,9 +50,14 @@ def __init__(self, parent=None):
         # Could also use a QTextEdit and set self.editor.setAcceptRichText(False)
         self.editor = QtWidgets.QPlainTextEdit()
 
-        # Load existing mss_settings.json
-        readMe = open(MSS_CONFIG_PATH + "/mss_settings.json", 'r').read()
-        self.editor.insertPlainText(readMe)
+        # Load mss_settings.json (if already exists)
+        self.mss_path = fs.path.join(MSS_CONFIG_PATH, "mss_settings" + ".json")
+        if self.mss_path is not None:
```

Comment:
This if will never be false. self.mss_path is just the path name joined together.
You are thus still running into the issue you tried to fix with this PR.

You could use `fs_url_exists(self.mss_path)`, from fslib.utils

Reply:
Okay I see, thanks. Will change it.

Reply:
pyfilesystems main idea is to unify all these file operations independent the kind of "storage" 
The API is excatly the same if you open a container or a file or a network storage, some more infos 
https://reposhub.com/python/miscellaneous/PyFilesystem-pyfilesystem2.html

I guess this is the missing part ;)

>>> _test_dir = fs.open_fs(MSS_CONFIG_PATH)
>>> _test_dir.exists('mss_settings.json')

on normal operations we import constants from mslib.msui but for tests we do from mslib._tests.

That makes than for tests the changes needed to have an independent path.

Reply:
Yes I did that in the latest commit :)